src/AlpineSkiHouse.Web/Events/PassPurchase.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+
+namespace AlpineSkiHouse.Events
+{
+    public class PassPurchase
```

Comment:
Maybe I approached this the wrong way. There is a PurchaseCompleted event which contains a number of PassPurchases. Would the individual line items also be named as events? Maybe instead I should be raising a number of individual PassPurchased events...